Get current camera parameters

ezaquarii opened the pull request.

ezaquarii commented:
This solution follows established practice and implements async API to obtain current camera parameters.

dmitry-zaitsev commented on fotoapparat/src/main/java/io/fotoapparat/hardware/v1/Camera1.java:
```diff
@@ -218,6 +220,18 @@ public void updateParameters(Parameters parameters) {
         parametersOperator().updateParameters(parameters);
 
         cachedZoomParameters = null;
+        currentParameters = parameters;
```
Those parameters will contain only those values which you are updating. For example, if you are updating only flash mode it will not contain picture size.

Actually, the whole thing can be done a bit simpler, without the need for a latch. You can use `camera.getParameters()` to get current parameters of the camera and then map them to our `Parameters` object (by adding this functionality to `ParametersConverter`).

ezaquarii commented:
1. Now we read camera parameters directly from Camera1.
2. Added converter to convert from platform to library parameters.
3. Refactored converter names.
4. Added converter test.

dmitry-zaitsev commented:
Now looks good. Thanks for contributing!
